tripy/nvtripy/backend/api/executable.py

```diff
@@ -299,6 +299,43 @@ def add(a, b):
         """
         json_utils.save(self, path)
 
+    def tensorrt_engine(self) -> bytes:
+        """
+        Returns the TensorRT engine from the executable if there's a valid one.
```

Comment:
There should always be a valid one right? If there isn't it's a Tripy/MLIR-TRT bug